### PR TITLE
enable ENS domain in URL

### PR DESCRIPTION
**Pull request by 0xlws**
This PR enables #65. It works when network is set to Ethereum Mainnet

### CODE REVIEW

File: components/Conversation/RecipientControl.tsx [hunk 1]
```diff
@@ -122,7 +125,7 @@ const RecipientControl = ({
           <AddressInput
             recipientWalletAddress={recipientWalletAddress}
             id="recipient-field"
-            className="block w-[95%] pl-7 pr-3 pt-[3px] md:pt-[2px] md:pt-[1px] bg-transparent caret-n-600 text-n-600 placeholder-gray-500 focus:outline-none focus:placeholder-gray-400 focus:ring-0 focus:border-transparent text-lg font-mono"
```
Comment (neekolas) [inline components/Conversation/RecipientControl.tsx:125]: Maybe these styling changes snuck into the branch. They don't feel related to the ultimate goal of the PR.

File: components/Layout.tsx [hunk 2]
```diff
@@ -36,12 +36,20 @@ const TopBarLayout: React.FC = ({ children }) => (
 )
 
 const ConversationLayout: React.FC = ({ children }) => {
+  console.log('ConversationLayout')
```
Comment (neekolas) [inline components/Layout.tsx:39]: Think these debugging logs snuck in as well

File: components/Layout.tsx [hunk 3]
```diff
@@ -67,6 +88,7 @@ const ConversationLayout: React.FC = ({ children }) => {
 }
 
 const Layout: React.FC = ({ children }) => {
+  console.log('Layout')
```
Comment (neekolas) [inline components/Layout.tsx:91]: Another debugging log we probably don't need in main

**Comment by bhavya2611** [comment]
I think this has been fixed after the latest changes, if you still see the need for the same we can reopen this PR. Closing it for now

**Pull request closed without merging**